ui/src/writeData/components/WriteDataItem.tsx

```diff
@@ -0,0 +1,70 @@
+// Libraries
+import React, {FC} from 'react'
+import {connect} from 'react-redux'
+import {withRouter, RouteComponentProps} from 'react-router-dom'
+
+// Components
+import {SelectableCard, SquareGrid, ComponentSize} from '@influxdata/clockface'
+
+// Utils
+import {getOrg} from 'src/organizations/selectors'
+
+// Graphics
+import placeholderLogo from 'src/writeData/graphics/placeholderLogo.svg'
+
+// Types
+import {WriteDataItem} from 'src/writeData/constants'
+import {AppState} from 'src/types'
+
+// Constants
+import {ORGS} from 'src/shared/constants/routes'
+
+// Styles
+import 'src/writeData/components/WriteDataItem.scss'
+
+interface StateProps {
+  orgID: string
+}
+
+type Props = WriteDataItem & RouteComponentProps & StateProps
+
+const WriteDataItem: FC<Props> = ({id, name, url, image, history, orgID}) => {
```

Comment:
use `useSelector` and `useHistory` here instead off connect and withRouter

Reply:
Yarrrr captain hook